app/src/main/java/ch/epfl/sweng/eventmanager/ui/event/interaction/fragments/schedule/ScheduleEditFragment.java

```diff
@@ -12,6 +12,7 @@
 import androidx.constraintlayout.widget.ConstraintLayout;
 import androidx.fragment.app.Fragment;
 import androidx.lifecycle.ViewModelProviders;
+import androidx.room.Ignore;
```

Comment:
Why this import ?

Reply:
Don't know why It didn't self optimized imports  ;)